api/alert_profiles.go

```diff
@@ -0,0 +1,139 @@
+//
+// Author:: Darren Murray (<darren.murray@lacework.net>)
+// Copyright:: Copyright 2022, Lacework Inc.
+// License:: Apache License, Version 2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package api
+
+import (
+	"fmt"
+
+	"github.com/pkg/errors"
+)
+
+// AlertProfilesService is the service that interacts with
+// the AlertProfiles schema from the Lacework APIv2 Server
+type AlertProfilesService struct {
+	client    *Client
+	Templates AlertTemplatesService
```

Comment:
AlertTemplates does not have a GET so cannot implement with our V2Service interface.

The api path is nested under AlertProfiles.
/api/v2/AlertProfiles/{alertProfileId}/AlertTemplates